packages/devextreme/js/ui/drop_down_editor/ui.drop_down_list.js

```diff
@@ -487,11 +487,8 @@ const DropDownList = DropDownEditor.inherit({
         this.setAria('owns', this._popup && this._popupContentId);
     },
 
-    _setDefaultAria: function() {
-        this.setAria({
-            'haspopup': 'listbox',
-            'autocomplete': 'list'
-        });
+    _getAriaHasPopup() {
+        return 'listbox';
```

Comment:
I understand that u only do a refactoring here, but I see that test for this is missed. Could u please additionally add it?